hazelcast/src/test/java/com/hazelcast/internal/networking/nio/AdvancedNetworkIntegrationTest.java

```diff
@@ -76,6 +78,31 @@ public void testCompleteMultisocketConfig() {
         assertWrongProtocolAlert(MEMCACHE_PORT, Protocols.CLUSTER, "AAA");
     }
 
+    @Test
+    @Category(QuickTest.class)
+    public void testTPCPortsWithAdvancedNetwork() {
+        Config config = smallInstanceConfig();
+        config.getTpcEngineConfig().setEnabled(true).setEventloopCount(4);
+        ServerSocketEndpointConfig clientSSConfig = new ServerSocketEndpointConfig();
+        clientSSConfig.getTpcSocketConfig()
+                .setReceiveBufferSize(1 << 20)
+                .setSendBufferSize(1 << 19)
+                .addPortDefinition("14000-" + (14000 + 4 - 2))
+                .addPortDefinition("15000-16000");
+        config.getAdvancedNetworkConfig().setEnabled(true).setClientEndpointConfig(clientSSConfig);
+
+        HazelcastInstance[] hz = new HazelcastInstance[3];
+        for (int i = 0; i < 3; i++) {
+            hz[i] = newHazelcastInstance(config);
+        }
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
Store this in a local variable so you don't need to traverse the path. Makes reason the test more complicated than needed.

Reply:
https://github.com/hazelcast/hazelcast/pull/23254/commits/a35381e11d97f32ac62932748b2d34a5c4208dc1